src/System Application/App/AppSource Gallery/src/AppSourceProduct.Table.al

```diff
@@ -0,0 +1,113 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Apps.AppSource;
+
+/// <summary>
+/// Represents an except of a product definition in AppSource.
+/// </summary>
+table 2515 "AppSource Product"
+{
+    DataClassification = SystemMetadata;
+    Access = Internal;
+    TableType = Temporary;
+
+    InherentEntitlements = RIMDX;
+    InherentPermissions = RIMDX;
```

Comment:
Strictly speaking, temporary tables don't need entitlements and permissions. But I don't think it hurts.